src/test/resources/features/account/import_account.feature

```diff
@@ -0,0 +1,21 @@
+@Account
+Feature: Import Accounts
+
+  @deleteAccountImports
+  Scenario: Import multiples accounts using csv files
+    Given I log in to Salesforce with Account Owner User credentials
+    When I go to ACCOUNTS tab
+    And I selected "Import" option
+    Then "Import your Data into Salesforce" should be displayed at title
+    And I selected "Accounts and Contacts"
+    And I selected "Add new records"
+    And I selected "Email" option from "Match Contact by:" field
```

Comment:
We have to write single steps, avoiding to have too many steps, these could be joined as only one step.